ADBench/run-all.ps1

```diff
@@ -51,7 +51,10 @@ param(# Which build to test.
       [double]$time_limit=10,  
       
       # Kill the test after this many seconds 
-      [double]$timeout=300,    
+      [double]$timeout=300,
+
+      # Kill the test if it consumes more than this many gigabytes of RAM
+      [double]$max_memory_amount_in_gb=8,
```

Comment:
Maybe increase this?
Do we know how many and which tests need more memory?

Reply:
Actually, I don't know. But I can increase it up to 16 GB (my machine performs all basical tests with this RAM size). Will it be enough?

Reply:
Another option is not to check RAM usage at all by default

Reply:
Yes, I think that would be the right thing to do.
In my dream we would have defaulted this value to something like total system memory - 2 GB, but .NET Core does not have the necessary API at the moment.

Reply:
RAM checking by default is turned off